modules/payments/public/schema/transaction.yml

```diff
@@ -2,6 +2,9 @@ name: transaction
 properties:
   - name: payable_ids
     type: array
+  - name: user_id
```

Comment:
Maybe it should be `creator_id` without defining the target type. There might be usecase when you assign it to some `record`